docs/scanners/pattern_search.md

```diff
@@ -15,8 +15,11 @@ scanner_configs:
         forbidden: true
         exclude_directory:
           - node_modules
-        exclude_extension:
```

Comment:
Can we use exclude and include extension simultaneously for different file types or can you only exclude or only include? Can you add a note in the documentation which one takes precedence in case a file extension is both included and excluded?

Reply:
No problem! Will update to more explicitly describe these behaviors. You can use both - while the [documentation](https://sift-tool.org/docs) isn't explicit about precedence, the [specs](https://github.com/coinbase/salus/pull/55/files#diff-57b4177c5dd615aad3f26daaca52e3f9R285) show that exclusion takes precedence over inclusion.